src/hardware/configuration.rs

```diff
@@ -211,15 +211,15 @@ pub fn setup(
         // Configure the timer to count at the designed tick rate. We will manually set the
         // period below.
         timer5.pause();
+        timer5.reset_counter();
         timer5.set_tick_freq(design_parameters::TIMER_FREQUENCY);
 
         // The timestamp timer must run at exactly a multiple of the sample timer based on the
         // batch size. To accomodate this, we manually set the prescaler identical to the sample
         // timer, but use a period that is longer.
         let mut timer = timers::TimestampTimer::new(timer5);
 
-        let period = digital_input_stamper::calculate_timestamp_timer_period();
-        timer.set_period_ticks(period);
+        timer.set_period_ticks(u32::MAX);
```

Comment:
I was under the impression we wanted the timestamp timer to overflow at a power-of-two at-least greater than the sampling timer period: https://github.com/quartiq/stabilizer/issues/170


> Timestamping timer overflow/reload period is j*n*t. Locked through the CPU PLL to the CPU XO. Not yet/necessarily deterministic (across reboots, resets, timer reconfigurations) phase w.r.t. ADC/DAC sampling timer. j >= 1, a power of two, as large as possible within the timer counter size.


Given my understanding of the timestamp timer now, I actually don't think we really care, since we internally manage roll-overs for phase calculation and setting the period to u32::MAX should support all of our use cases without issue. That being said, would you mind updating the comment on 217-220 appropriately?

Reply:
#170 is make obsolete by the implementation and documentation of the PLLs. We want and strictly need a i32 overflowing timer.